tests/src/CMakeLists.txt

```diff
@@ -32,9 +34,15 @@ foreach(programSourceFile ${programSourceFiles})
     message(STATUS "Skipping test ${programName}, since dune-iga were not found")
     continue()
   endif()
-  dune_add_test(SOURCES ${programSourceFile} LINK_LIBRARIES ikarus LABELS cpp)
-  target_compile_features(${programName} PUBLIC cxx_std_20)
-  set_tests_properties(${programName} PROPERTIES TIMEOUT 1000)
+  if(${programName} IN_LIST TEST_NEED_MORE_TIME)
+    dune_add_test(SOURCES ${programSourceFile} LINK_LIBRARIES ikarus LABELS cpp;cpp_slow)
```

Comment:
No wait this is not the way we discussed at the end I think at least.

So for the average programmer locally its better to have `cpp;cpp_quick`for the lower ones and `cpp`for the slow ones.